userspace/falco/outputs.proto

```diff
@@ -50,6 +50,5 @@ message response {
   string output = 5;
   map<string, string> output_fields = 6;
   string hostname = 7;
-  // TODO(leodido,fntlnz): tags not supported yet, keeping it for reference.
-  // repeated string tags = 8;
+  repeated string tags = 8;
```

Comment:
Maybe this should have been `optional` in the protobuf def?

Reply:
AFAIK, a `repeated` field is inherently `optional`.

Moreover, the protobuf compiler does not accept both modifiers simultaneously, for example:

```
optional repeated string tags = 8;
```

will produce a syntax error:

```
optional repeated string tags = 8;
```

Unless I'm missing something, we cannot add `optional` (and we don't need it).

Reply:
I endorse the comment of @leogr. From my understanding, the behavior of `repeated` is quite similar to `optional` already.